make.go

```diff
@@ -183,29 +183,31 @@ func makeUpstreamSourceTarball(gopkg string) (string, string, map[string]bool, s
 		}
 	}
 
-	// Remove all packages which are in the standard lib.
-	args := []string{"list", "-f", "{{.ImportPath}}: {{.Standard}}"}
-	args = append(args, godependencies...)
-
-	cmd = exec.Command("go", args...)
-	cmd.Stderr = os.Stderr
-	cmd.Env = []string{
-		fmt.Sprintf("GOPATH=%s", tempdir),
-		fmt.Sprintf("PATH=%s", os.Getenv("PATH")),
-	}
-	out, err = cmd.Output()
-	if err != nil {
-		return "", "", dependencies, autoPkgType, err
-	}
+	if len(godependencies) > 0 {
```

Comment:
You could make this pull request smaller, easier to review and the resulting code more elegant by using the following equivalent code, right?

```go
if len(godependencies) == 0 {
  return tempfile, version, dependencies, autoPkgType, nil
}
```